flyteidl/protos/flyteidl/core/workflow.proto

```diff
@@ -283,4 +283,6 @@ message TaskNodeOverrides {
     // Overrides for all non-standard resources, not captured by
     // v1.ResourceRequirements, to allocate to a task.
     ExtendedResources extended_resources = 2;
+
```

Comment:
could we add a comment here?

Reply:
Of course. How about https://github.com/flyteorg/flyte/pull/4858/commits/66fe2ca2f483f9bb0d381eb938f722253fb6a8e8?